src/components/screens/monitor/delegates/overview/forgingDetails.js

```diff
@@ -15,17 +15,9 @@ import styles from './overview.css';
 
 const FORGERS_TO_SHOW = 6;
 
-const getForgingStats = (data, forgedInRound) => {
-  if (forgedInRound === 0) return [0, 103, 0];
-  const states = {
-    forging: 0,
-    awaitingSlot: 0,
-    missedBlock: 0,
-  };
-  data.forEach((item) => {
-    states[item.state] += 1;
-  });
-  return [states.forging, states.awaitingSlot, states.missedBlock];
+const getForgingStats = (data, forgedInRound = 0) => {
+  const missedBlock = data.filter(item => item.state === 'missedBlock').length;
+  return [forgedInRound, ROUND_LENGTH - forgedInRound, missedBlock];
```

Comment:
Would be great if you can rename this to `missedBlocks`, it better represents what it holds